plaid/institutions.go

```diff
@@ -27,7 +27,8 @@ type Institution struct {
 }
 
 type InstitutionStatus struct {
-	ItemLogins ItemLogins `json:"item_logins"`
+	ItemLogins          ItemLogins                     `json:"item_logins"`
+	TransactionsUpdates InstitutionTransactionsUpdates `json:"transactions_updates"`
```

Comment:
Add entries for the other statuses (see [docs](https://plaid.com/docs/#institution-status))

Reply:
Done.